### ISSUE TITLE
I can't find the installer for ubuntu

### ISSUE THREAD

**Issue by glichtenstein** [devrkel/IPRStats#2]
Hello, 

I can't find the installer for ubuntu nor the downloads section, please advice.

A debian installer is provided at http://github.com/devrkel/IPRStats in the download section. Download the installer and double click it to open aptitude which will download and install any missing dependencies you may have. The installer will add a menu item under the Science section in Gnome.

Thank you in advance,

Best,

Gabriel

**Comment by cement-head** [comment]
There is no "Download" section

**Comment by glichtenstein** [comment]
Indeed, no download section as stated in the INSTALL file but a colleague pointed me to its url: 

https://github.com/devrkel/IPRStats/downloads

Case closed. 

Thanks.

**Comment by cement-head** [comment]
Yeah, that is weird that there is no link on the mainproject page

https://github.com/devrkel/IPRStats/downloads

**Comment by idoerg** [comment]
https://github.com/devrkel/IPRStats/downloads

There is no Ubuntu installer. you can download the Debian installer and
install it with dpkg

On Fri, Feb 26, 2016 at 9:39 AM, cement-head notifications@github.com
wrote:

> Yeah, that is weird that there is no link on the mainproject page
> 
> https://github.com/devrkel/IPRStats/downloads
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/devrkel/IPRStats/issues/2#issuecomment-189326404.

## 

Iddo Friedberg
http://iddo-friedberg.net/contact.html
++++++++++[>+++>++++++>++++++++>++++++++++>+++++++++++<<<<<-]>>>>++++.>
++++++..----.<<<<++++++++++++++++++++++++++++.-----------..>>>+.-----.
.>-.<<<<--.>>>++.>+++.<+++.----.-.<++++++++++++++++++.>+.>.<++.<<<+.>>

> > ----.<--.>++++++.<<<<------------------------------------.

**Comment by DeepakNandi** [comment]
> Yeah, that is weird that there is no link on the mainproject page
> 
> https://github.com/devrkel/IPRStats/downloads

even this webpage is not working
![iprstats_not_found](https://github.com/devrkel/IPRStats/assets/157372746/5fc031c3-6cf5-45bb-ae31-7e53f21b00da)
